src/FuelClient_TEST.cc

```diff
@@ -1366,6 +1366,52 @@ TEST_F(FuelClientTest, Models)
   }
 }
 
+//////////////////////////////////////////////////
+TEST_F(FuelClientTest, ModelsCheckCached)
+{
+  ClientConfig config;
+  std::string cacheDir = common::joinPaths(PROJECT_BINARY_PATH, "test_cache");
```

Comment:
this needs to be fixed since `PROJECT_BINARY_PATH` is no longer in use

Reply:
good call, I just amended to https://github.com/gazebosim/gz-fuel-tools/pull/362/commits/cfa86f7eaf84eb5b63a334651a4a8d90bb710989